### ISSUE TITLE
Fix contrast

### ISSUE THREAD

**Issue by Spaceface16518** [Spaceface16518/website#26]
The contrast in the "Read other posts" button is causing all accessibility tests to fail.

This should be fixed in the `user-choice-theme` branch (#18) and then merged into master. It can then be pulled on top of other branches.
